### PR TITLE
Major DV docs overhaul and migrating info from Classic DV/PT

### PR DESCRIPTION
**Pull request by martinkrulltott**
As the current DV docs is very brief this acts as a first draft for a major overhaul of the docs plus migration of info from the Classic DV and Classic PT apps.

FYI: Link to a useful .md VS Code plugin: https://marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced

### CODE REVIEW

**Comment by martinkrulltott** [comment]
`data-visualizer.md` currently contains 8 instances of `TODO: NEEDS INFO`. These are parts which I need help writing. Please see individual comments for each one.

File: src/commonmark/en/content/user/data-visualizer.md [hunk 1]
```diff
@@ -2,54 +2,323 @@
...
+<td><p>Radar</p></td>
+<td><p>Displays data on axes starting from the same point. Also known as spider chart.</p></td>
+</tr>
+<tr class="odd">
+<td><p>Gauge</p></td>
+<td><p>Semi-circle which displays values out of 100 %.</p></td>
+</tr>
+<tr class="even">
+<td><p>Year over year (line)</p></td>
+<td><p>TODO: NEEDS INFO </p></td>
```
Comment (martinkrulltott) [inline src/commonmark/en/content/user/data-visualizer.md]: This + the next 3 needs a short one-liner to give a brief explanation of what it is and what it's good for. Please see the other types above for an example of the format. @janhenrikoverland would you mind helping me with these?

File: src/commonmark/en/content/user/data-visualizer.md [hunk 2]
```diff
@@ -2,54 +2,323 @@
...
+
+#### Select periods
+
+When selecting a Period you have to option to choose between fixed periods and relative periods. These can also be combined. Overlapping periods are filtered so that they only appear once. For relative periods the names are relative to the current date, e.g. if the current month is March and **Last month** is selected, the month of February is added.
+
+![](resources/images/data-visualizer/data-visualizer-period-dimension-modal.png)
+
+#### Select organisation units
+
+TODO: NEEDS INFO
```
Comment (martinkrulltott) [inline src/commonmark/en/content/user/data-visualizer.md]: The Classic docs has this part: https://github.com/dhis2/dhis2-docs/blob/DV-add-from-classic-DV/src/commonmark/en/content/user/classic-data-visualizer.md#select-organisation-units, however I believe most of this has changed with the current org unit selector. I'm not fully aware of how this works, maybe @janhenrikoverland or @edoardo could help me here?

File: src/commonmark/en/content/user/data-visualizer.md [hunk 3]
```diff
@@ -2,54 +2,323 @@
...
+
+![](resources/images/data-visualizer/data-visualizer-organisation-unit-dimension-modal.png)
+
+## Change the display of your visualization
+
+TODO: NEEDS INFO, all options
```
Comment (martinkrulltott) [inline src/commonmark/en/content/user/data-visualizer.md]: @edoardo , would you mind doing this part? Following the same format as the other sections (examples above and  below). Here are the classic options for reference https://github.com/dhis2/dhis2-docs/blob/DV-add-from-classic-DV/src/commonmark/en/content/user/classic-data-visualizer.md#change-the-display-of-your-chart and classic PT https://github.com/dhis2/dhis2-docs/blob/DV-add-from-classic-DV/src/commonmark/en/content/user/analyze-data-in-pivot-tables.md#change-the-display-of-your-pivot-table

File: src/commonmark/en/content/user/data-visualizer.md [hunk 4]
```diff
@@ -2,54 +2,323 @@
...
+</tr>
+<tr class="even">
+<td><p>CSV</p></td>
+<td>Click <strong>CSV</strong></td>
+<td><p>Downloads CSV format based on the <strong>ID</strong>, <strong>Code</strong> or <strong>Name</strong> property.</p>
+</tr>
+<tr class="odd">
+<td><p>XML data value set</p></td>
+<td><p>Click <strong>Advanced > XML</strong></p></td>
+<td><p>TODO: NEEDS INFO</p></td>
```
Comment (martinkrulltott) [inline src/commonmark/en/content/user/data-visualizer.md]: @janhenrikoverland , would you know what this export (and the one below) does? Not sure how to explain this.

File: src/commonmark/en/content/user/data-visualizer.md (hunk 3 above)
Comment (edoardo) [reply src/commonmark/en/content/user/data-visualizer.md]: Working on it.

Comment (edoardo) [reply src/commonmark/en/content/user/data-visualizer.md]: Added the list with all available options for all vis types.
For the moment they are simply listed all together, with the descriptions ported and adapted from the old documentation.

**Review by jenniferarnesen — APPROVED** [review]
LGTM.

I've read through the entire document and made a few changes (ones you would expect from a pedant like myself).

File: src/commonmark/en/content/user/data-visualizer.md (hunk 3 above)
Comment (martinkrulltott) [reply src/commonmark/en/content/user/data-visualizer.md]: @edoardo changed the order and separated them by each tab here: https://github.com/dhis2/dhis2-docs/pull/410/commits/22bd2d2c33f19aa27be130cf04e3fa34fdc3153c, resolved!

**Comment by martinkrulltott** [comment]
All `TODO: NEEDS INFO` addressed and conversations resolved. Ready to merge @Philip-Larsen-Donnelly / @janhenrikoverland

**Pull request merged**